README.md

```diff
@@ -272,17 +272,110 @@ Take a look also at [vmagent](https://docs.victoriametrics.com/vmagent.html)
 and [vmalert](https://docs.victoriametrics.com/vmalert.html),
 which can be used as faster and less resource-hungry alternative to Prometheus.
 
-## Grafana setup
+## Setup VictoriaMetrics datasource plugin for Grafana
```

Comment:
It is a bad idea to change title because it forms a link. It makes old links incompatible. We shouldn't do this.